cmp/grid/Grid.ts

```diff
@@ -38,19 +50,8 @@ import {GridModel} from './GridModel';
 import {columnGroupHeader} from './impl/ColumnGroupHeader';
 import {columnHeader} from './impl/ColumnHeader';
 import {RowKeyNavSupport} from './impl/RowKeyNavSupport';
-import {RecordSet} from '@xh/hoist/data/impl/RecordSet';
-import {Icon} from '@xh/hoist/icon';
-
-import type {
-    ColDef,
-    ColGroupDef,
-    GetContextMenuItemsParams,
-    GridOptions,
-    GridReadyEvent,
-    ProcessCellForExportParams
-} from '@xh/hoist/kit/ag-grid';
 
-export interface GridProps extends HoistProps<GridModel>, LayoutProps {
+export interface GridProps extends HoistProps<GridModel>, LayoutProps, TestSupportProps {
```

Comment:
Could this extend `BoxProps`? (or if we make `BoxComponentProps` generic, could this extend `BoxComponentProps<GridModel>`)?